src/ACMESharp/Protocol/Messages/CreateAccountRequest.cs

```diff
@@ -10,8 +10,7 @@ namespace ACMESharp.Protocol.Messages
     /// </summary>
     public class CreateAccountRequest
     {
```

Comment:
Is this legal?  (To submit an Create Account request with no contact information.)

Reply:
Hi @ebekker, yes you can do this at some ACME servers when you create an account using EAB credentials. The RFC also notes that the field is optional:

https://tools.ietf.org/html/rfc8555#section-7.3